NewSpring/corinth
# chore: Live Button Plays in Native Player

## DESCRIPTION

### What does this PR do, or why is it needed?
This PR updates the Live Button so that it plays the live video in the native player.

### How do I test this PR?
Make a schedule live and tap on the live banner. It should open the video in the native player.

## TODO


## REVIEW


**Manual QA**


---

> The purpose of PR Review is to _improve the quality of the software._
